ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/defendant/CCDDefendant.java

```diff
@@ -90,4 +91,9 @@
     private LocalDate ccjRequestedDate;
 
     private CCDClaimantResponse claimantResponse;
+
+    private String reDeterminationExplaination;
```

Comment:
spelling mistake in the field, but its not your fault. Let we ask Sanchita whether she wants to change it not

Reply:
We need to fix this in definition and then here.

Reply:
fixed every where